presto-spark-base/src/main/java/com/facebook/presto/spark/execution/shuffle/PrestoSparkLocalShuffleInfoFactory.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution.shuffle;
+
+import com.facebook.presto.spark.classloader_interface.PrestoSparkShuffleReadDescriptor;
+import com.facebook.presto.spark.classloader_interface.PrestoSparkShuffleWriteDescriptor;
+import com.facebook.presto.spark.execution.PrestoSparkLocalShuffleReadInfo;
+import com.facebook.presto.spark.execution.PrestoSparkLocalShuffleWriteInfo;
+
+import java.util.Collections;
+
+public class PrestoSparkLocalShuffleInfoFactory
+        implements PrestoSparkShuffleInfoFactory<PrestoSparkLocalShuffleWriteInfo, PrestoSparkLocalShuffleReadInfo>
+{
+    private static final String LOCAL_SHUFFLE_ROOT_PATH = "/tmp";
```

Comment:
Can we create a temporary directory inside /tmp instead of hard code it? We can use Files.createTempDirectory()